lib/internal/Magento/Framework/Stdlib/Cookie/PhpCookieManager.php

```diff
@@ -70,17 +82,20 @@ class PhpCookieManager implements CookieManagerInterface
      * @param CookieReaderInterface $reader
      * @param LoggerInterface $logger
      * @param HttpHeader $httpHeader
+     * @param ScopeConfigInterface $scopeConfig
      */
     public function __construct(
         CookieScopeInterface $scope,
         CookieReaderInterface $reader,
         LoggerInterface $logger = null,
-        HttpHeader $httpHeader = null
+        HttpHeader $httpHeader = null,
+        ScopeConfigInterface $scopeConfig
```

Comment:
Adding a required constructor parameter is a backwards incompatible change, which may affect classes extending this one. Please check code example in [Backwards Compatible Development Guide](http://devdocs.magento.com/guides/v2.1/ext-best-practices/extension-coding/backwards-compatible-development/index.html#adding-a-constructor-parameter) and fix accordingly.